core/templates/dev/head/common-js-files.html

```diff
@@ -0,0 +1,33 @@
+<script src="/templates/head/app.js"></script>
```

Comment:
I'm not sure I understand how this is meant to work in prod. I thought we were going to have a prefix that changes based on whether we're running in the dev or prod env?